src/rprocessing/RLangPApplet.java

```diff
@@ -214,8 +214,9 @@ protected void wrapProcessingVariables() {
         this.renjinEngine.put("displayHeight", displayHeight);
         this.renjinEngine.put("focused", focused);
         this.renjinEngine.put("keyPressed", keyPressed);
-        this.renjinEngine.put("frameCount", frameCount);
+        this.renjinEngine.put("frameCount", (double) frameCount);
```

Comment:
Great. Maybe open an issue for this (Approaches to float-double casting) if there isn't one already.